pkg/scaffold/internal/templates/v2/main.go

```diff
@@ -48,6 +53,23 @@ func (f *Main) SetTemplateDefaults() error {
 	return nil
 }
 
+var hasher = fnv.New32a()
```

Comment:
Can this not be a local variable in `hash`?

Reply:
That would mean that we are creating a `Hash` every time.

Reply:
Is creating a `Hash` an expensive operation?

Reply:
[No](https://golang.org/src/hash/fnv/fnv.go?s=1113:1137#L33).

Reply:
No need to reset it then